checks/migrations/0016_webtestappsecpriv_timestamp_webtesttls_timestamp.py

```diff
@@ -0,0 +1,25 @@
+# Generated by Django 4.2.16 on 2024-12-03 22:43
+
+import datetime
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("checks", "0015_add_rpki_scoring"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="webtestappsecpriv",
+            name="timestamp",
+            field=models.DateTimeField(auto_now_add=True, default=datetime.datetime(1, 1, 1, 0, 0)),
```

Comment:
Will auto_now_add also apply to existing rows at the time of migration? Or will it only do that when adding a new object through ORM? Otherwise, we will be expiring currently existing reports early.

Reply:
This should set the date to epoch(0) for existing records. I will test it out to make sure.

Reply:
Yes but that's bad, right? If the cleanup runs right after deployment, it'll consider every report old, and start cleaning.

Reply:
The cleanup does not touch single test reports, only probe/test results. It will delete results that don't have a report associated and are not recent enough to be currently running. The timestamp is added to these 2 models because they don't have it yet while other tests in the report do.

Reply:
👍🏻